format-translators-module/format-translators-module/parser/structure_to_triples_parser.hpp

```diff
@@ -0,0 +1,44 @@
+#pragma once
+
+#include "parser/types.hpp"
+
+#include <sc-memory/sc_memory.hpp>
+
+namespace formatTranslators
+{
+class StructureToTriplesParser
+{
+public:
+  void ParseStructure(
+      ScMemoryContext * memoryContext,
+      ScAddr const & structureToTranslateAddr,
+      ScAddrToValueUnorderedMap<size_t> const & keyElementsOrderMap,
+      ScAddr const & identifiersLanguageAddr,
+      Triples & structureTriplesMap);
+
+private:
+  ScMemoryContext * context = nullptr;
```

Comment:
It is customary to specify a prefix `m_` for all class fields

Reply:
google codestyle and sc-machine codestyle don't have that rule